README.md

```diff
@@ -3,30 +3,49 @@ Thanks for interviewing at Stripe. To make sure that we can use our time best in
 
 We've noticed that more often than not, we've had to spend a significant portion of the interview helping candidates setup a Cmake-based C++ environment on their personal laptops.
 
-We've created a (rather ad-hoc) Cmake project to help candidates determine if their laptops were setup to write C++, so interviews can be about evaluating the candidate, and not the way their environment is setup.
+We've created a (rather ad-hoc) CMake project to help candidates determine if their laptops were setup to write C++, so interviews can be about evaluating the candidate, and not the way their environment is setup.
 
 # Getting ready
-First, clone this repository to your computer (creating a fork of the repository is not necessary). Next, ensure that you have `cmake` and a C++ compiler installed.
+First, clone this repository to your computer (creating a fork of the repository is not necessary). Next, ensure that you have `cmake`, a C++ compiler and `libcurl` installed.
 
+## Linux or macOS
 - If you're using Debian or Ubuntu, you likely want to run `sudo apt-get install cmake build-essential`.
-- If you're using macOS with HomeBrew installed, you might want to run `brew install cmake`.
-- Otherwise, you can find installation instructions for `cmake` [here](https://cmake.org/install/).
+- If you're using macOS with HomeBrew installed, you might want to run `brew install cmake curl`.
 
 If you can run these commands, your development environment is probably ready for Stripe's C++ interview questions:
 
 ```
-# On all platforms:
 $ cmake --version
 $ git clone [the project's URL]
 $ cd [into the project]
 $ mkdir build
 $ cd build
 $ cmake ..
-
-# On Linux or macOS:
 $ make
 $ ./hello
-
-# On Windows with Visual Studio installed:
-# open cpp-interview-prep.sln, build and run
 ```
+
+## Windows (Visual Studio)
+- Make sure you have the **C++ CMake tools for Windows** installed. These should automatically be there if you selected C++ as a language when installing Visual Studio. There are more instructions on installing `CMake` in Visual Studio [here](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=vs-2019#installation).
```

Comment:
Do we know if this will work on earlier versions of VS? Please don't spend a lot of time trying to figure this out.

Reply:
A quick Google search led me to the following link which suggests they've had support at least for a couple of years now. https://devblogs.microsoft.com/cppblog/cmake-support-in-visual-studio-whats-new-in-2017-15-2-update/

Also I used the Visual Studio Community Edition which is free to download so it isn't limited to paid versions.

Reply:
Sounds great.